src/lib/chat/matrix-client.ts

```diff
@@ -868,6 +881,7 @@ export class MatrixClient implements IChatClient {
 
   private mapConversation = async (room: Room): Promise<Partial<Channel>> => {
     const otherMembers = this.getOtherMembersFromRoom(room).map((userId) => this.mapUser(userId));
+    const memberHistory = this.getMemberHistoryFromRoom(room).map((userId) => this.mapMemberForHistory(userId));
```

Comment:
As we discussed, let's modify the user presence fetching to be more asynchronous (not delay the conversation rendering) and then we can use the `mapUser` here.

Hacked possible pseudo code
```
  // const chatClient = yield call(chat.get);
  //   for (let user of users) {
  //     const matrixId = user?.matrixId;
  //     if (!matrixId) continue;
  //     const presenceData = yield call([chatClient, chatClient.getUserPresence], matrixId);
  //     if (!presenceData) continue;
  //     const { lastSeenAt, isOnline } = presenceData;
        //  yield put(receiveUser({userId: ..., lastSeenAt, isOnline}))
  //   }
```